runtime/compiler/optimizer/J9Optimizer.cpp

```diff
@@ -283,6 +287,7 @@ static const OptimizationStrategy coldStrategyOpts[] =
    { OMR::trivialDeadTreeRemoval,                                               },
    { OMR::cheapTacticalGlobalRegisterAllocatorGroup, OMR::IfAOTAndEnabled            },
    { OMR::jProfilingValue,                           OMR::MustBeDone                 },
+   { OMR::treeLowering,                              OMR::MustBeDone            },
```

Comment:
I wasn't sure here whether it would be better to do lowering after JProfiling or before. I put it afterward for now but it should be easy enough to change later on if needed.